client/core/core.go

```diff
@@ -1933,18 +1956,37 @@ func (c *Core) initialize() {
 		wg.Add(1)
 		go func(acct *db.AccountInfo) {
 			defer wg.Done()
+			if acct.Disabled {
+				return
+			}
```

Comment:
How about not launching a goroutine for this `acct` instead (just do this check and `continue` before `wg.Add(1)`).

Reply:
I will try to make this change, thanks.